[ADD] accounting: new doc about outstanding account (#1887)
odoo/documentation: content/applications/finance/accounting/bank/setup/outstanding_accounts.rst

====================
Outstanding Accounts
====================

In Odoo, before being recorded in your bank account, payments are registered through transitory

> accounts, named Outstanding Accounts.

Reviewer: ```suggestion
accounts, named :guilabel:`Outstanding Accounts`.
```
@jcs-odoo Is this guideline already in place? (send your mail)

Reviewer 2: > @jcs-odoo Is this guideline already in place? (send your mail)

yes, but still not online, sorry :(

> - The **Outstanding Payments Account** is where outgoing payments are posted until they are linked

Reviewer: ```suggestion
- The :guilabel:`Outstanding Payments Account` is where outgoing payments are posted until they are linked
```

  to a withdrawal from your bank statement.
- The **Outstanding Receipts Account** is where incoming payments are posted until they are linked
  with a deposit from your bank statement.
These accounts should be of type "Current Asset".

.. note::
   The movement from an Outstanding Account to a bank account is done automatically when you
   reconcile the bank account with a bank statement.

Configuration
=============

To configure your Default Outstanding Accounts from the Accounting app, go to the
:menuselection:`Configuration --> Settings --> Default Accounts` and define your Outstanding
Receipts or Payments Account.

You can also decide to use specific Outstanding Accounts related to a specific journal.

> To do so, go to the :menuselection:`Dashboard --> Bank --> Configuration`. In the Incoming/Outgoing
> Payments tab, click on the dropdown column option to see and update your Outstanding accounts.

Reviewer 3: ```suggestion
To do so, go to your Dashboard, click the three vertical dots menu under Bank and click on Configuration. In the Incoming/Outgoing
Payments tab, click on the dropdown column option to see and update your Outstanding Accounts.
```

- It took me a few minutes to understand where I had to go with "Dashboard --> Bank --> Configuration". I know you don't want to use the word 'vertical ellipsis, but "three dots menu" or "three vertical dots menu" should do the trick and I understand using the menuselection directive with something like "Bank: three dots menu" might to be very clear (that's why I suggest putting that in text form)
- Issue: there is only a single Payments Configuration tab on v14. It seems it's where you need to set your default accounts. It's also quite different from v15.
- Capital A for "Outstanding Accounts" to be consistent

====================
Outstanding Accounts

> ====================

Author: dans ce cas, il faudrait également updater le titre de Bank account?! 
![image](https://user-images.githubusercontent.com/85342080/168537863-44f4f301-8b5d-4196-a2ef-7cb66e52d3bc.png)

> ====================

Author: Je veux bien qu'on reclarifie la règle concernant les sentence case... c'est pas super clair pour moi dans quel cas on le fait ou pas :)

In Odoo, before being recorded in your bank account, payments are registered through transitory
accounts, named Outstanding Accounts.

- The :guilabel:`Outstanding Payments Account` is where outgoing payments are posted until they are
  linked to a withdrawal from your bank statement.
- The :guilabel:`Outstanding Receipts Account` is where incoming payments are posted until they are
  linked with a deposit from your bank statement.

These accounts should be of type :guilabel:`Current Assets`.

.. note::
   The movement from an Outstanding Account to a bank account is done automatically when you
   reconcile the bank account with a bank statement.

Configuration
=============

The Outstanding Accounts are defined by default. If necessary, you can update them by going to
:menuselection:`Configuration --> Settings --> Default Accounts` and define your Outstanding
Receipts and Payments Accounts.

You can also decide to use specific Outstanding Accounts related to a specific journal.

To do so, go to your :guilabel:`Dashboard`, click the menu selection ⋮ under :guilabel:`Bank` and
click on :guilabel:`Configuration`. In the :guilabel:`Incoming/Outgoing Payments` tab, click on the

> dropdown column option to see and update your Outstanding Accounts.

Author: If I am right, it is only possible for the Bank or Cash journals.

> .. image:: outstanding_accounts/outstanding_acc_column.png

Reviewer 2: - separators = hyphen (and not underscore)
- no need to write "outstanding accounts" since it's already in the the right folder 

```suggestion
.. image:: outstanding_accounts/toggle-button.png
```

   :align: center

>    :alt: Select the dropdown column option and click on Outstanding Accounts

Reviewer 2: ```suggestion
   :alt: Select the toggle button and click on Outstanding Accounts
```

.. note::
   If you do not specify an Outstanding Payments Account or an Outstanding Receipts Account for a

>    specific journal, Odoo will use the default Outstanding Accounts.

Reviewer 2: ```suggestion
   specific journal, Odoo uses the default Outstanding Accounts.
```

before being recorded in your bank account.

- An **outstanding payments account** is where outgoing payments are posted until they are linked
  with a withdrawal from your bank statement.
- An **outstanding receipts account** is where incoming payments are posted until they are linked
  with a deposit from your bank statement.

These accounts should be of :ref:`type <chart-of-account/type>` :guilabel:`Current Assets`.

.. note::
   The movement from an outstanding account to a bank account is done automatically when you
   reconcile the bank account with a bank statement.

Configuration
=============

Default accounts configuration
------------------------------

The outstanding accounts are defined by default. If necessary, you can update them by going to
:menuselection:`Accounting --> Configuration --> Settings --> Default Accounts` and update your
:guilabel:`Outstanding Receipts Account` and :guilabel:`Outstanding Payments Account`.

Bank and cash journals configuration
------------------------------------

You can also set specific outstanding accounts for your :guilabel:`Bank` or :guilabel:`Cash`

> journals.

Reviewer 2: or any journal with the type bank or cash. Not only these default accounts ^^

From your :guilabel:`Accounting Dashboard`, click on the menu selection ⋮ of the journal you want to
configure, and click on :guilabel:`Configuration`, then open the :guilabel:`Incoming/Outgoing

> Payments` tab. To display the outstanding accounts column, click on the toggle button and check the

Reviewer 2: not sure if it's better with or without "the" before the next line.

> :guilabel:`Outstanding Receipts/Payments accounts`, then update the account.

Reviewer 2: (and Save? not sure it's _that_ important to mention but we usually do)
